proxysql-scheduler/local/test-binary-proxysql-scheduler

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#
+# Execute this tool to test binary releases
+#
+#  possible jenkins vars:
+#      MTR_ARGS
+#      MTR_REPEAT
+#      PXC_TARBALL
+
+set -o errexit
+set -o xtrace
+
+function process_mtr_output {
+# parameters are:
+  local TESTSUITE=${1}
+  local INFILE=${2}
+  local OUTFILE=${3}
+  egrep -a -o '[[:alpha:]]+(_[[:alnum:]]+)?(\.\/)?.*\[ .* \].*$' ${INFILE} | sort | \
+  gawk 'BEGIN { print "<testsuite name=\"@@TESTSUITE@@\" tests=\"@@PASSED@@\" failures=\"@@FAILURES@@\" disabled=\"@@DISABLED@@\" skipped=\"@@SKIPPED@@\">" }
+  /\[.*pass \]/ { printf "\t<testcase name=\"%s\" time=\"%.3f\"></testcase>\n", $1, $NF/1000.0 }
+  /\[ disabled \]/ { printf "\t<testcase name=\"%s\"><skipped/></testcase>\n", $1 }
+  /\[ skipped \]/  { printf "\t<testcase name=\"%s\"><skipped/></testcase>\n", $1 }
+  /\[.*fail \]/    { printf "\t<testcase name=\"%s\"><failure/></testcase>\n", $1 }
+  END { print "</testsuite>" }' > ${OUTFILE}.tmp
+  gawk '!seen[$0]++' < ${OUTFILE}.tmp > ${OUTFILE}
+  rm -f ${OUTFILE}.tmp
+#
+  PASSED=$(grep -c 'time=' ${OUTFILE} ||:)
+  FAILURES=$(grep -c '<failure/>' ${OUTFILE} ||:)
+  DISABLED=$(grep -c '\[ disabled \]' ${INFILE} ||:)
+  SKIPPED=$(grep -c '\[ skipped \]' ${INFILE} ||:)
+#
+  sed -i.bak -e "s:@@TESTSUITE@@:${TESTSUITE}:g" ${OUTFILE}
+  sed -i.bak -e "s:@@PASSED@@:${PASSED}:g" ${OUTFILE}
+  sed -i.bak -e "s:@@FAILURES@@:${FAILURES}:g" ${OUTFILE}
+  sed -i.bak -e "s:@@DISABLED@@:${DISABLED}:g" ${OUTFILE}
+  sed -i.bak -e "s:@@SKIPPED@@:${SKIPPED}:g" ${OUTFILE}
+  sed -i.bak -e "s: \[::g" ${OUTFILE}
+}
+
+
+# Setup ProxySQL
+sudo apt update
+
+pushd /tmp
+    sudo apt install -y lsb-release
+    sudo wget -O - 'https://repo.proxysql.com/ProxySQL/repo_pub_key' | sudo apt-key add -
+    sudo echo deb https://repo.proxysql.com/ProxySQL/proxysql-2.1.x/$(lsb_release -sc)/ ./ \ | sudo tee /etc/apt/sources.list.d/proxysql.list
+popd
+sudo apt update
+sudo apt install -y proxysql
```

Comment:
it will work only on deb based systems.
Why we use upstream package instead of ours?

Reply:
done